CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.5.2 (September 14, 2016)
+
+* BUG: The ec2-snapper `delete` command now gracefully exits if no snapshots exist for the given instance rather than
+  exiting with an error.
+
```

Comment:
It may be worth standardizing on adding a `CHANGELOG.md` to every repo, rather than relying solely on release notes. I find this a pretty efficient way to track changes. Thoughts?